run.py

```diff
@@ -28,7 +28,7 @@
 from pilates.beam import preprocessor as beam_pre
 from pilates.beam import postprocessor as beam_post
 from pilates.utils.io import read_datastore
-# from pilates.polaris.travel_model import run_polaris
+from pilates.polaris.travel_model import run_polaris
```

Comment:
@jaauld14 I had to comment this line out because the `polaris.travel_model` module imported a package called `polarisruntime` which I don't have access to. I'm guessing the containerized polaris will eventually replace this call but in the meantime if we're going to merge this code into master it has to work for everyone.